### PR TITLE
fixing up things

### PR DESCRIPTION
**Pull request by gioleppe**
I've changed the code in calculator.py, added unit tests in the root folder.

### CODE REVIEW

File: 1_lab/calculator.py [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
+        for el in range(abs(n)):
+            m -= 1
+    return m
 
 
 def divide(m, n):
-    result = 0
-    while m > 0:
-        result = +1
+    if m == 0 or n == 0:
```
Comment (vincenzopalazzo) [inline 1_lab/calculator.py:14]: This could be only ` n== 0`

**Review by vincenzopalazzo — COMMENTED** [review]
Thanks for this PR :-)

File: 1_lab/calculator.py (hunk 1 above)
Comment (gioleppe) [reply 1_lab/calculator.py:14]: fixing it, you're right

**Comment by vincenzopalazzo** [comment]
Thanks :-)

**Comment by gioleppe** [comment]
you're welcome :)

**Pull request merged**